DesignDoc.md

```diff
@@ -60,7 +62,7 @@ I will be using API Gateway and Lambda to create 14 API endpoints.
 I will be using tables in DynamoDB to store the player, group, game, and gameOutcome data.
 
 
-![GameBoard PlantUML](GameBoardCD.png)
+![GameBoard PlantUML](GameBoard CD.png)
```

Comment:
This isn't rendering for me on GitHub, I suspect it's because of the space in the filename. Spaces in directories/filenames are a pain to deal with anyways, I'd rename that file so there's no space in it.